[enterprise-4.12] add CoreDNS bump updates (#54095)
openshift/openshift-docs: release_notes/ocp-4-12-release-notes.adoc

For more information, see xref:../support/troubleshooting/troubleshooting-network-issues.html#overriding-default-node-ip-selection-logic_troubleshooting-network-issues[Optional: Overriding the default node IP selection logic]. 

[id="ocp-4-12-ne-coredns-bump"]
==== CoreDNS update to version 1.10.0

> In {product-title} {product-version}, CoreDNS uses version 1.10.0 which includes the following changes:

Reviewer: ```suggestion
In {product-title} {product-version}, CoreDNS uses version 1.10.0, which includes the following changes:
```

> * CoreDNS will not expand the query UDP buffer size if it was previously set to a smaller value.

Reviewer: ```suggestion
* CoreDNS does not expand the query UDP buffer size if it was previously set to a smaller value.
```

> * CoreDNS will now always prefix each log line in Kubernetes client logs with the associated log level.

Reviewer: ```suggestion
* CoreDNS now always prefixes each log line in Kubernetes client logs with the associated log level.
```
